circleci: failed tests: TestNodeRestart

The following test appears to have failed:

[#21055](https://circleci.com/gh/cockroachdb/cockroach/21055):

```
I160806 18:30:10.577080 acceptance/chaos_test.go:177  client 0 shutting down
I160806 18:30:10.582105 acceptance/chaos_test.go:413  1031 177.8/sec
I160806 18:30:10.682461 acceptance/cluster/localcluster.go:671  stopping
I160806 18:30:10.682661 acceptance/cluster/localcluster.go:571  event stream done, resetting...: net/http: request canceled
I160806 18:30:10.682681 acceptance/cluster/localcluster.go:594  events monitor exits
--- FAIL: TestNodeRestart (10.66s)
	cluster.go:74: roachpb.Batch RPC failed: rpc error: code = 13 desc = connection error: desc = "transport: read tcp 127.0.0.1:45356->127.0.0.1:32804: read: connection reset by peer"
=== RUN   TestContinuousLoad_BlockWriter
--- SKIP: TestContinuousLoad_BlockWriter (0.00s)
	util_test.go:357: skipping since not run against remote cluster
=== RUN   TestContinuousLoad_Photos
--- SKIP: TestContinuousLoad_Photos (0.00s)
	util_test.go:357: skipping since not run against remote cluster
=== RUN   TestEventLog
I160806 18:30:12.581328 acceptance/cluster/localcluster.go:299  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:357: skipping since not run against remote cluster
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/chaos_test.go
+++ b/acceptance/chaos_test.go
@@ -246,7 +246,9 @@ func chaosMonkey(state *testState, c cluster.Cluster, stopClients bool, pickNode
 			time.Sleep(time.Second)
 		}
 		c.Assert(state.t)
-		cluster.Consistent(state.t, c)
+		// TODO(peter): Disabled until https://github.com/grpc/grpc-go/pull/818 (or
+		// similar) goes in.
+		// cluster.Consistent(state.t, c)
 		log.Warningf(context.Background(), "round %d: cluster recovered", curRound)
 	}
 }

